.github/workflows/codeowners.yml

```diff
@@ -12,6 +12,15 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
+      - name: Check indentation
+        run: |
+          no_different_indentations=$(cat .github/CODEOWNERS | grep -v '^#' | awk NF | sed s/@.*//g | awk '{ print length }' | sort | uniq | wc -l)
```

Comment:
spent an hour or so trying to write a prettier parser. after ripping all the hairs off my head i decided bash was good enough.